abandoned_battlefield/map.conf

```
@@ -0,0 +1,13 @@
+initial_stuff = default:cobble 40, shooter:pistol, default:pick_steel
```

It's an unloaded pistol...
```suggestion
initial_stuff = default:cobble 40, shooter:pisto_loadedl, default:pick_steel
```